tutorials/codegen.ipynb

```diff
@@ -480,66 +480,66 @@
     "        # scope: The subgraph of the state containing only the scope (map contents)\n",
     "        #        we want to generate the code for.\n",
-    "        # state_id: The state in the SDFG the subgraph is taken from (i.e., \n",
+    "        # state_id: The state in the SDFG the subgraph is taken from (i.e.,\n",
     "        #           `sdfg.node(state_id)` is the same as `scope.graph`)\n",
     "        # function_stream: A cursor to the global code (which can be used to define\n",
     "        #                  functions, hence the name).\n",
     "        # callsite_stream: A cursor to the current location in the code, most of\n",
     "        #                  the code is generated here.\n",
-    "        \n",
+    "\n",
     "        # We can get the map entry node from the scope graph\n",
     "        entry_node = scope.source_nodes()[0]\n",
-    "        \n",
+    "\n",
     "        # First, generate an opening brace (for instrumentation and dynamic map ranges)\n",
     "        callsite_stream.write('{', sdfg, state_id, entry_node)\n",
-    "        \n",
+    "\n",
     "        ################################################################\n",
-    "        # Generate specific code: We will generate a reversed loop with a \n",
+    "        # Generate specific code: We will generate a reversed loop with a\n",
     "        # comment for each dimension of the map. For the sake of simplicity,\n",
     "        # dynamic map ranges are not supported.\n",
-    "        \n",
+    "\n",
     "        for param, rng in zip(entry_node.map.params, entry_node.map.range):\n",
     "            # We use the sym2cpp function from the cpp support functions\n",
     "            # to convert symbolic expressions to proper C++\n",
     "            begin, end, stride = (sym2cpp(r) for r in rng)\n",
-    "            \n",
+    "\n",
     "            # Every write is optionally (but recommended to be) tagged with\n",
     "            # 1-3 extra arguments, serving as line information to match\n",
     "            # SDFG, state, and graph nodes/edges to written code.\n",
     "            callsite_stream.write(f'''// Loopy-loop {param}\n",
     "            for (auto {param} = {end}; {param} >= {begin}; {param} -= {stride}) {{''',\n",
     "                                  sdfg, state_id, entry_node\n",
     "            )\n",
-    "        \n",
+    "\n",
     "            # NOTE: CodeIOStream will automatically take care of indentation for us.\n",
-    "        \n",
-    "        \n",
+    "\n",
+    "\n",
     "        # Now that the loops have been defined, use the dispatcher to invoke any\n",
     "        # code generator (including this one) that is registered to deal with\n",
     "        # the internal nodes in the subgraph. We skip the MapEntry node.\n",
-    "        self.dispatcher.dispatch_subgraph(sdfg, scope, state_id,\n",
+    "        self.dispatcher.dispatch_subgraph(sdfg, dfg, scope, state_id,\n",
```

Comment:
Same as above, the argument is `cfg`: https://github.com/spcl/dace/blob/main/dace/codegen/dispatcher.py#L380